src/client/serve-placement.js

```diff
@@ -57,7 +63,7 @@ function servePlacement () {
       let remove
       const add = () => {
         remove = applyStyles(
-          'qubit-cli-placement-styles',
+          PLACEMENT_STYLE_ELEMENT_ID,
           require('placement.less')
```

Comment:
Wasn't it failing to successfully require the file here?

Reply:
Yeah it was, but I removed the loader so now `require('placement.less')` returns the CSS string after less compilation

Reply:
That's it, sorry bit thick